src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -176,6 +176,344 @@ namespace cryptonote
+        int count;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amounts)
+          KV_SERIALIZE(count)
+        END_KV_SERIALIZE_MAP()
+      };
+    
+      
+      struct output {
+        std::string public_key;
+        std::string global_index; //TODO: why is this a string in this call but not get_unspent_outs.
+        std::string rct;               
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(public_key)
+          KV_SERIALIZE(global_index)
+          KV_SERIALIZE(rct)                                 
+        END_KV_SERIALIZE_MAP()
+      };
+      
+      struct amount_out {
+        std::string amount;
+        std::vector<output> outputs;
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amount)
+          KV_SERIALIZE(outputs)                                
+        END_KV_SERIALIZE_MAP()
+        
+      };
+      
+      struct response
+      {
+        std::vector<amount_out> amount_outs;
+        std::string Error;
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amount_outs)
+          KV_SERIALIZE(Error)
+        END_KV_SERIALIZE_MAP()
+      };
+  };
+  //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_SUBMIT_RAW_TX
+  {
+      struct request
+      {
+        std::string address;
+        std::string view_key;
+        std::string tx;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key)
+          KV_SERIALIZE(tx)  
+        END_KV_SERIALIZE_MAP()
+      };
+    
+      
+      struct response
+      {
+        std::string status;
+        std::string error;
+        
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(status)
+          KV_SERIALIZE(error)
+        END_KV_SERIALIZE_MAP()
+      };
+  };
+  //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_LOGIN
+  {
+      struct request
+      {
+        std::string address;
+        std::string view_key;
+        bool create_account;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key) 
+          KV_SERIALIZE(create_account) 
+        END_KV_SERIALIZE_MAP()
+      };
+    
+      
+      struct response
+      {
+        std::string status;
+        std::string reason;
+        bool new_address;
+        
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(status)
+          KV_SERIALIZE(reason)
+          KV_SERIALIZE(new_address)
+        END_KV_SERIALIZE_MAP()
+      };
+  };
+  //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_IMPORT_WALLET_REQUEST
+  {
+      struct request
+      {
+        std::string address;
+        std::string view_key;
+        
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key)
+        END_KV_SERIALIZE_MAP()
+      };
+    
+      
+      struct response
+      {
+        std::string payment_id;
+        std::string import_fee;
```

Comment:
Same as amount in get_unspent_outs. MyMonero design. @vtnerd @fluffypony @paulshapiro